### DISCUSSION TITLE
Welcome to PaddleOCR Discussions!

### DISCUSSION THREAD

**Discussion by Evezerest** [PaddlePaddle/PaddleOCR#6007] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xiaoyangyang2** [comment]
I'm a developer who uses PaddleOCR to improve my productivity and I'm happy to see the community getting better and better with you all!
